src/plugins/liveobjects/livemap.ts

```diff
@@ -455,6 +450,37 @@ export class LiveMap<T extends API.LiveMapType> extends LiveObject<LiveMapData,
     return { update: { [op.key]: 'removed' } };
   }
 
+  /**
+   * Returns true if the origin timeserials of the given operation and entry indicate that
+   * the operation should be applied to the entry, following the CRDT semantics of this LiveMap.
+   */
+  private _canApplyOperationFollowingCRDTRules(
```

Comment:
Would a better name for this be `_canApplyMapOperation`?

Reply:
yes! I was struggling with naming this one 😅 , and just saying `Map` is better for sure, thanks!

Reply:
updated